packages/visualizations/addon/utils/table.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import keyBy from 'lodash/keyBy';
+
+export function indexColumnById(columns) {
```

Comment:
Can we add some unit tests for this?

Reply:
I like moving this logic into it's own function, but can we move it back into the `models/table.js` around here: https://github.com/yahoo/navi/blob/92e7e5e11934cefea7b05cad19bfae061a701d30/packages/visualizations/addon/models/table.js#L107